### DISCUSSION TITLE
Email provider on iOS: session-token not passed from gmail safari web view to safari browser app

### DISCUSSION THREAD

**Discussion by sbkl** [nextauthjs/next-auth#4524] [Help]
Hi,

Using the  email provider, when logging in on laptop, I get no issue.

However, when testing on iOS, I use the safari native app to request a login link. The link is sent by email to my gmail account.

Then, I go the gmail app to open the email and click on the link which opens a webview inside gmail (I select safari) and the login process works and I can access the app.

However, the session-token is not available when opening the same url to the safari native app.

Any idea of what I am missing?

Thank you.

**Comment by jperezrealini** [comment]
Also experiencing this issue.

Related:
- https://github.com/orgs/supabase/discussions/15708
- https://github.com/abevoelker/devise-passwordless/issues/61
